libbpf-rs/src/map.rs

```diff
@@ -476,6 +476,26 @@ impl Map {
             ptr: std::ptr::null_mut(),
         })
     }
+
+    /// Attach a struct ops map
+    pub fn attach_struct_ops(&mut self) -> Result<Link> {
+        if self.map_type() != MapType::StructOps {
+            return Err(Error::InvalidInput(format!(
+                "Invalid map type ({}) for attach_struct_ops()",
+                self.map_type(),
+            )));
+        }
+
+        let ret = unsafe {
```

Comment:
nit: would you mind renaming "ret" to "ptr" to make it more clear that it returns a link ptr?